src/panels/logbook/ha-logbook.ts

```diff
@@ -58,6 +62,18 @@ class HaLogbook extends LitElement {
     `;
   }
 
+  private async _fetchUsers() {
+    const users = await fetchUsers(this.hass);
+    users.forEach((user) => {
+      this._userid_to_name[user.id] = user.name;
```

Comment:
Other option would be to copy it: `this._userid_to_name = { ...this._userid_to_name, user.id: user.name };` But that is making a lot of copies.

Reply:
WE should make it a `@property, create a new object here and just assign it to `this._userid_to_name = userid_to_name`